client/src/containers/interventions/new/component.tsx

```diff
@@ -56,12 +80,24 @@ const InterventionForm = () => {
             >
               Cancel
             </button>
-            <button
-              type="submit"
-              className="ml-3 inline-flex justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-green-700 hover:bg-green-800 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
-            >
-              Continue
-            </button>
+            {!!currentStep && !isFirstStep && (
+              <button
+                type="submit"
+                className="ml-3 inline-flex justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-green-700 hover:bg-green-800 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
```

Comment:
This button should be a component that we can reuse due to the styles mainly, or at least the className should be exported to use it.